### ISSUE TITLE
Add a warning if the target "element" can't be found in the original source code

### ISSUE THREAD

**Issue by rbarrois** [novafloss/raincoat#17]
With version 0.8.0, if the ``# Raincoat`` comment points to an invalid element, there is no warning nor output.

For instance, with the following code:
```python
from django.conf import settings
from django.db.backends.oracle import base                                                                                             
    
    
class DatabaseOperations(base.DatabaseOperations):  # pylint: disable=abstract-method                                                  
        
    def _convert_field_to_tz(self, field_name, tzname):                                                                                
        # Raincoat: pypi package: Django==1.8.4 path: django/db/backends/oracle/operations.py element: _convert_field_to_tz            
```

Running ``raincoat`` leads to the following output:
```
% raincoat 
Collecting Django==1.8.4
  Using cached [....]/Django-1.8.4-py2.py3-none-any.whl
  Saved /tmp/tmpzrv6fd7r/Django-1.8.4-py2.py3-none-any.whl
Successfully downloaded Django
```

When I fix this to ``element: DatabaseOperations._convert_field_to_tz``, I receive the proper output:
```diff
% raincoat
Collecting Django==1.8.4
  Using cached https://.../Django-1.8.4-py2.py3-none-any.whl
  Saved /tmp/tmpxpg868d7/Django-1.8.4-py2.py3-none-any.whl
Successfully downloaded Django
Django == 1.8.4 vs 1.10.7 @ django/db/backends/oracle/operations.py:DatabaseOperations._convert_field_to_tz (from blusers/core/optiservice/oracle_optiservice/base.py:11)
Code is different:
--- django/db/backends/oracle/operations.py
+++ django/db/backends/oracle/operations.py
@@ -1,13 +1,14 @@
def _convert_field_to_tz(self, field_name, tzname):
-        if not self._tzname_re.match(tzname):
-            raise ValueError("Invalid time zone name: %s" % tzname)
-        # Convert from UTC to local time, returning TIMESTAMP WITH TIME ZONE.
-        result = "(FROM_TZ(%s, '0:00') AT TIME ZONE '%s')" % (field_name, tzname)
+        if settings.USE_TZ:
+            if not self._tzname_re.match(tzname):
+                raise ValueError("Invalid time zone name: %s" % tzname)
+            # Convert from UTC to local time, returning TIMESTAMP WITH TIME ZONE.
+            field_name = "(FROM_TZ(%s, '0:00') AT TIME ZONE '%s')" % (field_name, tzname)
# Extracting from a TIMESTAMP WITH TIME ZONE ignore the time zone.
# Convert to a DATETIME, which is called DATE by Oracle. There's no
# built-in function to do that; the easiest is to go through a string.
-        result = "TO_CHAR(%s, 'YYYY-MM-DD HH24:MI:SS')" % result
-        result = "TO_DATE(%s, 'YYYY-MM-DD HH24:MI:SS')" % result
+        field_name = "TO_CHAR(%s, 'YYYY-MM-DD HH24:MI:SS')" % field_name
+        field_name = "TO_DATE(%s, 'YYYY-MM-DD HH24:MI:SS')" % field_name
# Re-convert to a TIMESTAMP because EXTRACT only handles the date part
# on DATE values, even though they actually store the time part.
-        return "CAST(%s AS TIMESTAMP)" % result
+        return "CAST(%s AS TIMESTAMP)" % field_name
```

**Comment by ewjoachim** [comment]
Hmm, that's an interesting one :) According to [the code](https://github.com/novafloss/raincoat/blob/master/raincoat/match/python.py#L106-L110), an appropriate error should be displayed ! I'll add a test for that and fix it !

Thank you a lot for your report and I'm really glad to know there are people out there who use Raincoat !

**Comment by ewjoachim** [comment]
The root cause was that Raincoat saw the object did not exist both in the old code and the new code, and said, because it was identical, it was not worth reporting. Fixing it :)

**Comment by ewjoachim** [comment]
Fixed in 0.8.2, released on PyPI. Tell me if it works better for you !
